### ISSUE TITLE
journal logo keyboard focus contrast insufficient.

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5470]
1.9 RC 1 journal logo on clean and on clarity keyboard focus contrast insufficient

Material and OLH seem fine, e.g. Material:

<img width="320" height="232" alt="Image" src="https://github.com/user-attachments/assets/f5be44bc-a157-440d-b204-cb91af9a2d2e" />

But clean and clarity are not:

<img width="288" height="206" alt="Image" src="https://github.com/user-attachments/assets/71357918-bb7b-425b-841f-7ea4929f6400" />

<img width="251" height="170" alt="Image" src="https://github.com/user-attachments/assets/1c6b49ae-107d-4e7d-b986-1073cb98463c" />

Note: there's a lot of keyboard focus css in the files, please review carefully to see where to add these. I suspect the issue is that their element type has not been covered by all the other selectors as they are image links.

**Comment by ajrbyers** [comment]
Closed by https://github.com/openlibhums/janeway/pull/5521
